lib/wrapper.js

```diff
@@ -297,8 +297,8 @@ class RMLMapperWrapper {
   /**
    * Paths in the RML files can't have \\ so all / must be replaced by \, for windows, the mapper takes care of reverting this again.
    */
-  _fixPath(path) {
-    return path.replace(/\\/g, '/');
+  _fixPath(filePath) {
+    return path.sep === '\\' ? filePath.replace(/\\/g, '/') : filePath;
```

Comment:
The comment says "all / must be replaced by \", but this is not what is happening in the `replace`.